engine/npcyclemanager/update.go

```diff
@@ -27,6 +29,41 @@ func (db *DB) update(ctx context.Context, all bool, alertID *int) error {
 	}
 	log.Debugf(ctx, "Updating notification cycles.")
 
-	_, err = db.lock.Exec(ctx, db.queueMessages)
-	return errors.Wrap(err, "queue outgoing messages")
+	tx, err := db.lock.BeginTx(ctx, nil)
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
+
+	rows, err := tx.StmtContext(ctx, db.queueMessages).QueryContext(ctx)
+	if err != nil {
+		return err
+	}
+	defer rows.Close()
+
+	type record struct {
+		alertID int
+		userID  string
+		meta    *alertlog.NotificationMetaData
```

Comment:
Since this isn't used we can leave it out, just pass nil as the final arg to `LogTx`